manifest.json

```
@@ -3,8 +3,13 @@
   "name": "Active Yuwa",
   "version": "0.1.0",
   "description": "Ask Buddie extension to improve the post reach.",
+  "permissions": ["scripting", "activeTab", "storage"],
+  "host_permissions": [
+    "https://www.facebook.com/*",
+    "https://m.facebook.com/*",
+    "https://web.facebook.com/*"
```

We should only allow the extension in Ask Buddie group.
```suggestion
    "https://*.facebook.com/groups/askbuddie/*"
```